ruby_review_week_four.rb

```diff
@@ -2,36 +2,131 @@
 
 # 1. What is a variable?
 
+# varible is a place holder with name to store data.
+
 # 2. What Ruby method can you use to make a loop from an array?
 
+# .each
+
 # 3. In Ruby, what are booleans?
 
+# true and false.
+
 # 4. How do classes relate to objects?
 
+# objects are made out of classes. Class are the general idea of a thing and object is a thing.
+
 # 5. What is an instance variable?
 
+# variables that belong to object are called instance variables.
```

Comment:
yep this is correct, i also like the use of the word `attribute` when we're talking about instance variables 

> an instance variable is an attribute of an object